src/bindings/python/src/pyopenvino/graph/node_input.cpp

```diff
@@ -75,4 +75,8 @@ void regclass_graph_Input(py::module m) {
                 get_source_output : Output
                     Output that is connected to the input.
               )");
+    input.def_property_readonly("get_rt_info",
```

Comment:
It would be nice to have the possibility to modify rt_info as well. Can we align implementation with rt_info for node?
```
    node.def("get_rt_info",
             (PyRTMap & (ov::Node::*)()) & ov::Node::get_rt_info,
             py::return_value_policy::reference_internal,
             R"(
                Returns PyRTMap which is a dictionary of user defined runtime info.
                Returns
                ----------
                get_rt_info : PyRTMap
                    A dictionary of user defined data.
             )");
...
    node.def_property_readonly("rt_info",
                               (PyRTMap & (ov::Node::*)()) & ov::Node::get_rt_info,
                               py::return_value_policy::reference_internal);
```

Reply:
Sure we can try, I will update the implementation and add additional tests

Reply:
@nkogteva I tested it, and I am afraid that it is not possible to modify rt_info cause in python we are working with ConstOutput which in cpp is `Output<const ov::Node>` and get_rt_info underneath is calling ```m_node->get_rt_info``` but since node is a constant and if we try to modify RTMap like e.g.:
`rt_info = output_node.get_rf_info()`
`rt_info["test"] = "test"`
We are not modyfing the content of RTMap of output_node but rather modyfing the content of rt_info newly created object

Reply:
@bszmelcz I didn't catch what is the reason for the problem and why node rt_info is modifiable, but output rt_info is not. cpp API has several methods (including without const):
```
RTMap& Output<Node>::get_rt_info() {
    return m_node->m_outputs.at(m_index).get_rt_info();
}

const RTMap& Output<Node>::get_rt_info() const {
    return m_node->m_outputs.at(m_index).get_rt_info();
}

const RTMap& Output<const Node>::get_rt_info() const {
    return m_node->m_outputs.at(m_index).get_rt_info();
}
```

Reply:
Ok,  my comment might be misleading. There is no problem with exposing the `get_rt_info()`. The question that I have is if it is expected that Output<const Node> will not be able to update runtime info, which is what I encountered. From my point of view that is correct behavior since the underlying code of get_rt_info(), which you pasted above, is using m_node which is in fact  storing RTMap where runtime info is being stored and since m_node is a constant it seems reasonably that it cannot be modified. That is all that I wanted to clarify. Is it clear for you?

Reply:
I pushed the changes that you have requested so you can take a look if that is what you meant :)

Reply:
@bszmelcz  Could you please add tests with tf info modification?

Reply:
@nkogteva added